ebpfcore/ebpf_drv.c

```diff
@@ -143,7 +152,7 @@ _ebpf_driver_initialize_objects(
 
     device_initialize = WdfControlDeviceInitAllocate(
         *driver,
-        &SDDL_DEVOBJ_SYS_ALL_ADM_ALL // only kernel/system and administrators.
+        &security_descriptor // only kernel/system, administrators and ebpfsvc.
```

Comment:
"only kernel/system, administrators, and ebpfsvc."

Commo before "and" when enumerating items. :-)